winterize/init.lua

```diff
@@ -48,6 +48,10 @@ minetest.register_lbm({
 	nodenames = {"group:crumbly", "group:leaves"},
 	run_at_every_load = true,
 	action = function(pos, node)
+		if node.name == "default:clay" then -- Some floors in Karsthafen are made of clay
```

Comment:
It's not a good idea to hardcode stuff in the game on the basis of existing maps. The snow spawning code should be robust enough to figure this out on its own.

Reply:
I can't think of anything other than a raycast and hardcoding surface nodes

Reply:
Raycast isn't necessary for a top-down section. LVM is your friend, though. ;)

Reply:
Mod is now using LVMs